robusta_krr/core/integrations/kubernetes.py

```diff
@@ -190,7 +211,7 @@ async def list_clusters(self) -> Optional[list[str]]:
             return None
 
         try:
-            contexts, current_context = await asyncio.to_thread(config.list_kube_config_contexts)
+            contexts, current_context = config.list_kube_config_contexts()
```

Comment:
why did you remove the thread instead of doing something like this?
```
contexts, current_context = await loop.run_in_executor(
            self.executor, lambda: config.list_kube_config_contexts())
```

Reply:
Because this config listing is both fast and blocking, I descided that it does not need running in ThreadPool

Reply:
Async thread is waiting for `list_kube_config_contexts` to complete before continuing anything else (we await the `list_clusters`).
So it does not really matter if we run it in a threadpool